sox.features.js

```diff
@@ -685,14 +683,14 @@
             // Description: For adding some text to questions that are in the hot network questions list
 
             function addHotText() {
-                if (!$('.sox-hot').length) {
-                    $('#feed').html('<p>SOX: One of the 100 hot network questions!</p>');
+                if (!document.getElementsByClassName('sox-hot').length) {
+                    document.getElementById('feed').innerHTML = '<p>SOX: One of the 100 hot network questions!</p>';
 
                     //display:block to fix https://github.com/soscripted/sox/issues/243:
-                    $('#question-header').css('display', 'block').prepend('<div title="SOX: this is a hot network question!" ' + (sox.location.on('english.stackexchange.com') ? 'style="padding:13px"' : '') + ' class="sox-hot"><i class="fab fa-free-code-camp"></i><div>');
+                    $(document.getElementById('question-header')).css('display', 'block').prepend('<div title="SOX: this is a hot network question!" ' + (sox.location.on('english.stackexchange.com') ? 'style="padding:13px"' : '') + ' class="sox-hot"><i class="fab fa-free-code-camp"></i><div>');
```

Comment:
Is there any point in using vanilla JS just to make it a jQuery object straight after? :/

Reply:
Actually, `$(document.getElementById(ID))` is quite faster than `$("#ID")`...

Reply:
@GaurangTandon fair enough, I never knew that!